### PR TITLE
Features/token page/navmenu

### LINKED ISSUE #91: Implement the List of Token Sales Page
## Detailed Description

To implement the token listings re-design  for the Mesa MVP.
[Figma Redesign](https://www.figma.com/file/7vH0nZB72tizFbhT7gtizo/Mesa-MVP?node-id=408%3A170)

## Context
Bertie has completed this final design for the Mesa MVP therefore we can now start to implement these and I have decided to start with this.

### LINKED ISSUE #94: Implement Custom Page Navagation Link
## Detailed Description
Implement the custom button page navigation link to allow users to switch between upcoming, open and closed views of the Mesa MVP.

### PR DESCRIPTION
**Pull request by hhamud**
---
name: Pull Request & Code Review
about: Create a pull request for this project
title: complete nav bar menu

---

## Description
Closes #94 
Closes #91 
Created  `AuctionNavBar` Component
Created  `SaleBadge` Component
Created context provider for filtering between the auctions
Updated the Auctions index page 
Created Title

## Motivation and Context
This is a part of the auctions listings page implementation of bertie's design for the Mesa MVP

## How Has This Been Tested?
I have added tests for the components that I have created

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Review by adamazad — COMMENTED** [review]
Great work. Let's discuss a few conventions over Jitsu with @light-fury today so that the code base and quality is coherent among authors.

File: src/components/Container/index.tsx [hunk 1]
```diff
@@ -1,28 +1,29 @@
...
   inner?: boolean
   noPadding?: boolean
 }
 
+
+
 export const Container = styled.div<ContainerProps>(
   props => ({
-    position: 'absolute',
+    position: props.position === 'absolute' ? 'absolute': 'unset',
```
[the remark this answers is not in the document]
Comment (hhamud) [reply src/components/Container/index.tsx]: I will create another component that extends the extends the container component purely to use as the absolute container as you have mentioned in the code.

File: src/views/Auctions/components/AuctionSummaryCard/index.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    const auction: Auction = {
+      id: jest.fn(),
+      startBlock: 1646500442,
+      endBlock: 1678036442,
+      tokenIcon: jest.fn(),
+      tokenAddress: jest.fn(),
+      tokenSymbol: jest.fn(),
+      tokenAmount: jest.fn(),
+      tokenName: jest.fn(),
+      bids: jest.fn(),
```
[the remark this answers is not in the document]
Comment (hhamud) [reply src/views/Auctions/components/AuctionSummaryCard/index.spec.tsx:35]: I'll make sure the mocks match the properties for the tests

File: src/components/SaleBadge/index.tsx [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  position: relative;
+  margin-bottom: 48px;
+  display: flex;
+  height: 33px;
+  width: 240px;
+  border-radius: 33px;
+  background: #dddde3;
+`
+
+export const UnactiveTextBadge = styled.p<UnactiveTextBadgeProps>(
```
[the remark this answers is not in the document]
Comment (hhamud) [reply src/components/SaleBadge/index.tsx:19]: I'll get to refactoring that and have you review them for my next pull request

**Pull request merged**